TaskLayer/SearchTask/PostSearchAnalysisTask.cs

```diff
@@ -1017,59 +1030,220 @@ private void WriteVariantResults()
+            {
+                deletionSites += entry.Value.Count();
+            }
+
+            int frameshiftSites = 0;
+            foreach (var entry in frameshiftVariants)
+            {
+                frameshiftSites += entry.Value.Count();
+            }
+
+            int stopGainSites = 0;
+            foreach (var entry in stopGainVariants)
+            {
+                stopGainSites += entry.Value.Count();
+            }
+
+            int stopLossSites = 0;
+            foreach (var entry in stopLossVariants)
+            {
+                stopLossSites += entry.Value.Count();
+            }
+
+            int totalVariantSites = SNVmissenseSites + MNVmissenseSites+ insertionSites + deletionSites + frameshiftSites + stopGainSites + stopLossSites;          
+
+            
 
-            string[] variantResults = new string[15];
+            string[] variantResults = new string[25];
             variantResults[0] = "Variant Result Summary";
             variantResults[2] = "--------------------------------------------------";
-            variantResults[4] = "Number of potential variant containing peptides identified at 1% FDR: " + variantPeptides.Where(p => p.IsDecoy ==false &&  p.IsContaminant ==false && p.FdrInfo.QValue <= 0.01 &&p.FdrInfo.QValueNotch <= 0.01).ToList().Count();
-            variantResults[5] = "Number of unqiuely identified variant peptides at 1% FDR: " + confidentVariantPeps.Where(p => p.IsDecoy == false &&  p.IsContaminant == false && p.FdrInfo.QValue <= 0.01 && p.FdrInfo.QValueNotch <= 0.01).ToList().Count();
-            variantResults[6] = "Number of SAV variant peptides at 1% FDR: " + savCount;
-            variantResults[7] = "Number of frameshift variant peptides at 1% FDR: " + frameshiftCount;
-            variantResults[8] = "Number of inframe insertion variant peptides at 1% FDR: " + insertionCount;
-            variantResults[9] = "Number of inframe deletion variant peptides at 1% FDR: " + deletionCount;
-            variantResults[10] = "Number of stop gain variant peptides at 1% FDR: " + stopGainCount;
-            variantResults[11] = "Number of stop loss variant peptides at 1% FDR: " + stopLossCount;            
-            variantResults[12] = "Number of modified variant peptides at 1% FDR: " + modifiedVariantPeptides.Count();
-            variantResults[13] = "Number of modified variant sites at 1% FDR: " + modifiedVariantSitePeptides.Count();
+            variantResults[4] = "Number of potential variant containing peptides identified at 1% group FDR: " + variantPeptides.Where(p => p.IsDecoy ==false &&  p.IsContaminant ==false && p.FdrInfo.QValue <= 0.01 &&p.FdrInfo.QValueNotch <= 0.01).ToList().Count();
+            variantResults[5] = "Number of unqiuely identified variant peptides at 1% group FDR: " + confidentVariantPeps.Where(p => p.IsDecoy == false &&  p.IsContaminant == false && p.FdrInfo.QValue <= 0.01 && p.FdrInfo.QValueNotch <= 0.01).ToList().Count();
```

Comment:
`=> !p.IsDecoy && !p.IsContaminant` is a bit more typical

Check that you can't use  `.Count` property, not the `.Count()` method